src/shared/utils.py

```diff
@@ -25,3 +25,24 @@ def format_plan_nickname(product_name: str, plan_interval: str) -> str:
 
 def get_indempotency_key() -> str:
     return uuid.uuid4().hex
+
+
+def format_brand(brand: str) -> str:
```

Comment:
One thing is not like the other.  This seems to be in line with vendor utils, credit utils, etc and not so much like the `get_indempotency_key` above.  That is my one and only minor nitpick here.  This being said, I will approve it given the expediency to demo and we can figure it out post demo given it is only a movement of a function across files.

Reply:
Moved to vendor_utils.py